third_party/conan/docker/digests.sh

```diff
@@ -23,9 +23,9 @@ declare -rA docker_image_digest_mapping=(
   [gcc10_release]="gcr.io/orbitprofiler/gcc10_release@sha256:3479c24eaa2512618ecba1872743495583d531af0b5cbd7acf52bc73da54fbeb" \
   [gcc10_relwithdebinfo]="gcr.io/orbitprofiler/gcc10_release@sha256:3479c24eaa2512618ecba1872743495583d531af0b5cbd7acf52bc73da54fbeb" \
   [gcc10_debug]="gcr.io/orbitprofiler/gcc10_debug@sha256:3479c24eaa2512618ecba1872743495583d531af0b5cbd7acf52bc73da54fbeb" \
-  [ggp_release]="gcr.io/orbitprofiler/ggp_release@sha256:c3143f4fa3a503a8c2fe674f0efa82ec48623a3bfa9627feedf6e23de26d3454" \
```

Comment:
This looks a bit funny. Why did this change from ``ggp_debug`` to ``ggp_release``?

Looks the same for the other toolchains and the sha's are identical so I gues this is fine.

Reply:
It is probably based on the order which I used as arguments for the build/upload scripts?